libs/components/tiles/src/lib/modules/tiles/tile-dashboard/tile-dashboard.component.ts

```diff
@@ -76,44 +76,49 @@ export class SkyTileDashboardComponent implements AfterViewInit, OnDestroy {
   public configChange = new EventEmitter<SkyTileDashboardConfig>();
 
   @ViewChildren(SkyTileDashboardColumnComponent)
-  private columns: QueryList<SkyTileDashboardColumnComponent>;
+  public columns: QueryList<SkyTileDashboardColumnComponent> | undefined;
 
   @ViewChild('singleColumn', {
     read: SkyTileDashboardColumnComponent,
     static: false,
   })
-  public singleColumn: SkyTileDashboardColumnComponent;
+  public singleColumn: SkyTileDashboardColumnComponent | undefined;
 
-  public tileMovedReport: string;
+  public tileMovedReport = '';
 
-  public moveInstructionsId =
-    this.dashboardService.bagId + '-move-instructions';
+  public moveInstructionsId: string;
 
-  private _config: SkyTileDashboardConfig;
-
-  private configSet = false;
-
-  private ngUnsubscribe = new Subject<void>();
-
-  private viewReady = false;
+  #configSet = false;
+  #dashboardService: SkyTileDashboardService;
+  #mediaQueryService: SkyMediaQueryService;
+  #ngUnsubscribe = new Subject<void>();
+  #resourcesService: SkyLibResourcesService | undefined;
+  #viewReady = false;
+  #_config!: SkyTileDashboardConfig;
```

Comment:
We need to figure out a way to remove this non-null assertion because the config can technically be undefined at any moment. For example, what happens if the consumer wants to complete an asynchronous task before setting the config after `ngOnInit`? If any of this class's methods access the `config` value, they will throw an unhandled error. (From the looks of it, you're already adding `if` checks for the config anyway?)

Reply:
Making the config nullable seems to be ok. Made the change